stance/predict/overlay.py

```diff
@@ -13,7 +16,51 @@ def display_overlay(input_video, template_skeletons):
         path to the video of the user performing the motion
     template_skeletons : List[Skeletons]
     """
-    pass
+    cap = cv2.VideoCapture(input_video)
+    numFrames = int(cap.get(cv2.CAP_PROP_FRAME_COUNT))
+    width = int(cap.get(cv2.CAP_PROP_FRAME_WIDTH))
+    height = int(cap.get(cv2.CAP_PROP_FRAME_HEIGHT))
+
+    # used for scaling template skeleton to user video
+    scale_x = width / 1280
+    scale_y = height / 720
+    count = 0
+
+    for i in range(numFrames):
+        _, frame = cap.read()
+
+        # User skeleton
+        temp = list(user_skeletons[i].vectors.values())
```

Comment:
When I run the examples, this keeps throwing an index out of bounds?

Reply:
You could also inline this since you don't use it again, and then you have a different "temp" variable later

Reply:
Meant to push the fixed version. My bad. Also I added a try except that would make it awkward to do inline